tests/models/altclip/test_modeling_altclip.py

```diff
@@ -402,6 +402,15 @@ class AltCLIPModelTest(ModelTesterMixin, PipelineTesterMixin, unittest.TestCase)
     test_resize_embeddings = False
     test_attention_outputs = False
 
+    # TODO: Fix the failed tests when this model gets more usage
```

Comment:
All such changes in this PR is about:

  - pipeline tests fail for newly created tiny models (for recently added models)
  - pipeline tests fail for existing tiny models but not added to `tiny_model_summary.json` due to my ignorance in previous PR(s).

Reply:
For all of these, should we open issues to track them?